<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mashStepEditor</class>
 <widget class="QDialog" name="mashStepEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mash Step Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_name">
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_mashStepType">
         <property name="text">
          <string>Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_mashStepType</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BtComboBox" name="comboBox_mashStepType">
         <property name="toolTip">
          <string>Type</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="SmartLabel" name="label_stepTemp">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Target temp.</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_stepTemp</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_stepTemp">
         <property name="toolTip">
          <string>Target temp. of this step</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="SmartLabel" name="label_amount">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Infuse Amount</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_amount</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_amount">
         <property name="toolTip">
          <string>Amount of water to infuse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="SmartLabel" name="label_infuseTemp">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Infuse temp.</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_infuseTemp</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_infuseTemp">
         <property name="toolTip">
          <string>Temperature of infusion water</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="SmartLabel" name="label_stepTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_stepTime</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_stepTime">
         <property name="toolTip">
          <string>Time to conduct the step</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="SmartLabel" name="label_rampTime">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Temp. lag time</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_rampTime</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_rampTime">
         <property name="toolTip">
          <string>Lag time</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="SmartLabel" name="label_endTemp">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>End temp.</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_endTemp</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SmartLineEdit" name="lineEdit_endTemp">
         <property name="toolTip">
          <string>Final temp. of this step</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="toolTip">
        <string>New</string>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>Save and close</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="toolTip">
        <string>Discard and close</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/BtComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
</ui>
